src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/representer/BaseRepresenter.kt

```diff
@@ -205,8 +205,8 @@ abstract class BaseRepresenter(
                     val anyKeyOrValueNotPlain =
                         value.any { (keyNode, valueNode) ->
                             !(keyNode is ScalarNode && keyNode.isPlain)
-                                    ||
-                                    !(valueNode is ScalarNode && valueNode.isPlain)
+                                ||
```

Comment:
Let's avoid accidental style changes.

Reply:
I tried reverting it, but this is how IntelliJ autoformats it.

Reply:
Hmm, ideally we should agree on given style and somehow enforce it to avoid noise in PRs. Do you know if there's a tool that checks if code is formatted according to IntelliJ rules? Quodana perhaps?
(non-blocking in this PR, just wanted to discuss it for the future)

Reply:
We committed IDEA code style config into the repo - I suspect that this file just hasn't been reformatted in a while.

Reply:
I'm not a huge fan of format checkers, like ktlint. It's really frustrating to keep getting nagged about codestyle while I'm hacking around trying to fix something.

Reply:
Both approaches have pros and cons. We can meet in the middle: agree on some style and enforce it (best if IDEA style could be enforced), but create it as a separate CI check. Thanks to this, you won't be nagged when iterating fast locally, and still it's checked by CI and it's clear if just the styling is wrong. You won't have to fix formatting until the very last moment of submitting a PR for review. WDYT?